package.json

```diff
@@ -50,9 +50,13 @@
         "xmldoc": "^0.1.2",
         "grunt-cleanempty": "1.0.3"
     },
+    "devDependencies": {
+        "http-server": "^0.8.0"
+    },
     "scripts": {
         "postinstall": "grunt install",
-        "test": "grunt test && grunt build-browser-compressed"
+        "test": "grunt test && grunt build-browser-compressed",
+        "start": "http-server --port=8080 src"
```

Comment:
Let's make one change here.  The docs above indicate that you should browse to `http://localhost:8080/src`, and if we use `.` instead of `src` as our root, we'll be able to reuse this for both the `src/` and `dist/` cases (i.e., if you happen to have a built `dist/` and want to see it, you can just go to `http://localhost:8080/dist`).

Also, can you please change `8080` to `8000` since that is what the stock Thimble env uses when looking for this server, see https://github.com/mozilla/thimble.webmaker.org/blob/bramble/env.dist#L109-L110